addons/website/static/src/components/dialog/edit_menu.js

```diff
@@ -112,8 +112,7 @@ export class EditMenuDialog extends Component {
 
         this.menuEditor = useRef('menu-editor');
 
-        this.state = useState({ rootMenu: {} });
-
+        this.state = useState({rootMenu: {}, flatMenus: new Map()});
```

Comment:
I'm not sure I understand why this was necessary 😅 It looks like the state isn't used in the template, and every time we touch the map, we also change rootMenu (or one of its children) so a re-render is already triggered right?